### ISSUE TITLE
quirk: addresses in BTC wallet are displayed using cashaddr

### ISSUE THREAD

**Issue by molecular** [markblundeberg/openswap#7]
if you set "use cashaddr format" in preferences, cashaddr format is used even in BTC wallet.

obviously legacy format should be used always for BTC wallet regardless of setting in prefs.

**Comment by molecular** [comment]
in addition I just observed very confusing behaviour

  * went to my BTC wallet window
  * I unchecked "cashaddr" in tools / preferences
  * go to receive tab
  * observe "receiving address" is legacy format
  * BUT: observe: qr-code displayed is legacy format but qr-code still has "bitcoincash:" prefix

EDIT: Ooops, sorry. I see this is actually an upstream bug in electron-cash. Will open an issue there.

**Comment by markblundeberg** [comment]
Yeah, the cashaddr / legacy thing is unfortunately done with a global variable switch. So, it will be tough to support separate address format displays for BCH and BTC. Unfortunately, the only feasible solution right now seems to be disabling cashaddr support.

**Comment by molecular** [comment]
ok, I understand. Thanks for the info.
